.github/workflows/galaxy.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: Release to Ansible Galaxy
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: galaxy
+        uses: robertdebock/galaxy-action@1.0.1
+        with:
+          galaxy_api_key: ${{ secrets.galaxy_api_key }}
```

Comment:
You will need to create `galaxy_api_key ` in [secrets](https://github.com/brianshumate/ansible-consul/settings/secrets) and put there API Key from Ansible Galaxy https://galaxy.ansible.com/me/preferences